packages/seatmaps-client/package.json

```diff
@@ -20,7 +20,6 @@
   "license": "MIT",
   "main": "./cjs/index.js",
   "module": "./esm/index.mjs",
-  "types": "./cjs/index.d.ts",
```

Comment:
types property is still needed I believe for ts types to work unless It resolves automatically somehow, not sure about removing it.